kpatch-build/kpatch-build

```diff
@@ -602,6 +602,13 @@ echo "Building patch module: kpatch-$PATCHNAME.ko"
 cp "$OBJDIR/.config" "$SRCDIR"
 cd "$SRCDIR"
 make prepare >> "$LOGFILE" 2>&1 || die
+
+if [[ $DISTRO == ubuntu ]]; then
+  # UBUNTU: add UTS_UBUNTU_RELEASE_ABI to utsrelease.h after regenerating it
+  UBUNTU_ABI=$(echo "$ARCHVERSION" | awk '{split($1, a, "-"); print a[2]}')
```

Comment:
We'd need to do it twice since we want 'A' from X.Y.Z-A-flavor.
So either the one liner to awk, or:
```
UBUNTU_ABI=${ARCHVERSION#*-}
UBUNTU_ABI=${UBUNTU_ABI%-*}
```